### PR TITLE
feat: registerPage && chore: loginPage

### PR DESCRIPTION
**Pull request by ihsaanhardyanto**

### CODE REVIEW

File: src/controller/UserController.java [hunk 1]
```diff
@@ -47,10 +47,6 @@ public Map<String, String> registerUser(User user, String password) {
 	}
 
 	public User loginUser(String username, String password) {
-		if (username.equals("admin") && password.equals("admin")) {
-			return new User(0, "admin", "", "", "", "Admin");
-		}
-
```
Comment (raexera) [inline src/controller/UserController.java:53]: Why was this removed?

**Review by raexera — COMMENTED** [review]
Check my comment in your commit at src/controller/UserController.java

File: src/controller/UserController.java (hunk 1 above)
Comment (ihsaanhardyanto) [reply src/controller/UserController.java:53]: accidentally deleted

**Review by raexera — APPROVED** [review]
